Plugzy.Domain/Common/BaseEntity.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugzy.Domain.Common
+{
+    public class BaseEntity
```

Comment:
IEntity direk buna bağlayabilirsin .   ayrıca  baseentity içine eklediğin prop larıda buraya ekle,   appuser ı ayrıştırabilirsin bu noktada.   genericrepo  ile zaten  baseentity classına işlem yapıyor olacaksın.

user için  repo ya muhtemelen ihtiyacın olmayacak  identity nin usermanager vb onları kullanabilirsin